app/assets/v2/js/grants/info.js

```diff
@@ -39,6 +39,14 @@ function addToCart(grantData) {
         return;
     }
 
+    // Add donation defaults
+    // TODO Update to use real data
+    grantData.grant_donation_amount = 1;
+    grantData.grant_donation_currency = 'DAI';
+    grantData.grant_donation_type = 'one-time'; // options are 'one-time' and 'recurring'
+    grantData.grant_donation_num_rounds = 0; // N/A if type is one-time
```

Comment:
Based on my reading of the existing code, I believe a one time grant is just represented as `num_rounds = 1`, so I think we should follow this convention. That way we don't have to store a separate `grant_donation_type` variable, but can just infer it from the num rounds. That work?